src/shipit_workflow/shipit_workflow/tasks.py

```diff
@@ -50,9 +55,16 @@ def __init__(self, description):
         self.description = description
 
 
+def get_trust_domain(project):
+    if 'comm' in project:
+        return 'comm'
+    else:
+        return 'gecko'
+
+
 def find_decision_task_id(project, revision):
-    decision_task_route = 'gecko.v2.{project}.revision.{revision}.firefox.decision'.format(
-        project=project, revision=revision)
+    decision_task_route = '{trust_domain}.v2.{project}.revision.{revision}.taskgraph.decision'.format(
```

Comment:
Can you make sure to land the corresponding patch to maple, jamun and birch please.

Reply:
Done.